### PR TITLE
Duckfarts fix create tag page for nested

### PR DESCRIPTION
**Pull request by duckfarts**
Fix for **Creating a tag page for a nested tag doesn't work #20**
- Replaces the `/` symbols in tag names with `_` when creating filenames

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ export default class TagPagePlugin extends Plugin {
 		// Append # to tag if it doesn't exist
 		const tagOfInterest = tag.startsWith('#') ? tag : `#${tag}`;
 		const { isWildCard, cleanedTag } = getIsWildCard(tagOfInterest);
-		const filename = `${cleanedTag.replace('#', '')}${
+		const filename = `${cleanedTag.replace("#", "").replaceAll("/","_")}${
```
Comment (GerkinDev) [inline main.ts:145]: Might be even better if the replacement char were configurable via options. I typically use `>` to reflect imbrication

Comment (mjsumpter) [reply main.ts:145]: I don't believe Obsidian registers `#tag>imbrication` though. Only `#tag/imbrication`

Comment (GerkinDev) [reply main.ts:145]: I meant for the replacement char, instead of `_`

**Comment by mjsumpter** [comment]
@duckfarts @GerkinDev just want to call out that I see your PRs and very much appreciate them! Just need to find time to review and test them, hopefully this weekend 🤞

File: main.ts (hunk 1 above)
Comment (GerkinDev) [reply main.ts]: Indeed, they are apparently forbidden under Windows; but we only use linux distros, so if it isn't a valid char, the page creation will hang (just like it does currently).

https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file

So, worst case scenario depending on user choices: nothing changes compared to the current bug
Best case scenario: pimp that path

**Review by mjsumpter — APPROVED** [review]

**Pull request merged**